daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/Transaction.scala

```diff
@@ -392,9 +392,11 @@ object Transaction {
    *        time.
    */
   final case class Metadata(
+      submissionSeed: Option[crypto.Hash],
       submissionTime: Time.Timestamp,
       usedPackages: Set[PackageId],
-      dependsOnTime: Boolean
+      dependsOnTime: Boolean,
+      nodeSeeds: ImmArray[(Value.NodeId, crypto.Hash)],
```

Comment:
Maybe not for this PR: Do you intend to add the nodeSeeds also to `TransactionMeta` used in the `TransactionAccepted` update that's sent over the read service?

Reply:
Definitively not for this PR, but this kind of change is in the scope of #5500.